### PR TITLE
2.03 defin recipient region vocab uri 341

### PR DESCRIPTION
**Pull request by amy-silcock**
Added Extra Documentation as per: https://github.com/IATI/IATI-Schemas/issues/341

### CODE REVIEW

**Review by PetyaKangalova — APPROVED** [review]
Refers to change of definition for vocabulary uri and adds the correct Discuss link

**Review by PetyaKangalova — APPROVED** [review]

**Comment by dalepotter** [comment]
Changed base branch in order to force a refresh to the commit and file diff. This PR previously included changes already in the base branch.

File: en/activity-standard/iati-activities/iati-activity/transaction/recipient-region.rst [hunk 1]
```diff
@@ -25,6 +25,10 @@ Full example:
 Changelog
 ~~~~~~~~~
 
+2.03
```
Comment (dalepotter) [inline en/activity-standard/iati-activities/iati-activity/transaction/recipient-region.rst:28]: According to the [schema issue](https://github.com/IATI/IATI-Schemas/issues/341), `iati-activity/sector/@vocabulary-uri` is updated, not this one (`iati-activity/transaction/recipient-region/@vocabulary-uri`)

Comment (dalepotter) [reply en/activity-standard/iati-activities/iati-activity/transaction/recipient-region.rst:28]: The schema change to `iati-activity/transaction/recipient-region/@vocabulary-uri` has originally missed in issue IATI/IATI-Schemas#341 (now updated), with the associated fix in PR IATI/IATI-Schemas#411

Additionally:
- The changelog change for `iati-activity/sector/@vocabulary-uri` was made in https://github.com/IATI/IATI-Extra-Documentation/commit/743878709714d36636168b4f00ada6e1aa53f695#diff-7babd61f66c5b1d62ed7cb64b3d37f49 (PR #412)
- The changelog change for `iati-activity/transaction/sector/@vocabulary-uri` was made in https://github.com/IATI/IATI-Extra-Documentation/commit/743878709714d36636168b4f00ada6e1aa53f695#diff-f249553f1379d881cf17b0caa382f449 (PR #412)

This means this PR can be approved and merged.

**Review by dalepotter — APPROVED** [review]

**Pull request merged**